full-stack-asset-transfer-guide/applications/trader-typescript/src/contract.ts

```diff
@@ -76,8 +76,10 @@ export class AssetTransfer {
     }
 
     async transferAsset(id: string, newOwner: string, newOwnerOrg: string): Promise<void> {
-        // TODO: Implement me!
-        // Submit a 'TransferAsset' transaction, which requires [id, newOwner, newOwnerOrg] arguments.
+        console.log(`transferring asset '${id}' to ${newOwner}, ${newOwnerOrg}`);
```

Comment:
remind me to update the example contracts to use the proper logging system at some point...